src/pages/compare/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+
+import { Flex, Heading, VStack, Center, Spinner, HStack } from '@chakra-ui/react';
+import { useParams } from 'react-router';
+
+import { Timetable, useGetTimetable } from 'lib/fetchers';
+import { SavedCourse, useSavedCourses } from 'lib/hooks/useSavedCourses';
+
+import { Header } from 'common/header';
+
+import { TimetableActionButtons } from 'pages/import/components/TimetableActionButtons';
+import { SelectedCoursesCardList } from 'pages/scheduler/components/share/SelectedCoursesCardList';
+
+export function TimetableComparison(): JSX.Element {
+  const { slug } = useParams();
+  const [timetableCourses, setTimetableCourses] = useState<SavedCourse[]>([]);
+  const { courses } = useSavedCourses();
+
+  const { loading, data } = useGetTimetable({ slug: slug });
+
+  useEffect(() => {
+    if (!loading && data) {
+      const newCourseList: SavedCourse[] = (data as Timetable).courses.map((course) => {
```

Comment:
this is a bug - right now in the compare page _all_ the user's saved courses are shown, not just the ones that match the term of the timetable being imported, which makes it confusing

@keithradford should I not include these courses at all or should I have like headings and show all courses in each term?

Reply:
Solved by doing the following:
<img width="987" alt="image" src="https://user-images.githubusercontent.com/50898635/150725023-60c604b1-9561-41ff-a4ac-0650578c539b.png">

The UI will always show the section for the timetable term (either display courses or show no courses were found), and will show other terms conditionally (if the user has courses saved)